docs/dg/dev/third-party-module-development/distribute.md

```diff
@@ -0,0 +1,132 @@
+---
+title: How to Publish a Module on Packagist
+description: How to Publish a Module on Packagist
+last_updated: Jun 7, 2024
+template: howto-guide-template
+
+---
+
+## Prerequisites
+
+* Install PHP
+* Install composer
+* A Git repository for hosting your package (e.g., GitHub, GitLab, Bitbucket).
+
+## Create a PHP package
+
+1. Create a directory for your package:
+```bash
+mkdir my-package
+cd my-package
+```
+
+2. Add your PHP code to this directory, for example—create a file named `src/Example.php`.
```

Comment:
Are you referring to the module's code? Isn't that supposed to be in a specific directory at this point?

Reply:
Removed this part and just reffered the previously created repo